content/en/serverless/google_cloud_run_function_gen1/_index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Install Serverless Monitoring for Google Cloud Run Functions Gen 1
+---
+
+## Overview
+This page explains how to collect traces, trace metrics, runtime metrics, and custom metrics from your Cloud Run Functions Gen 1(formerly known as cloud functions) . For Gen 2 support please go to [this doc][8] and to collect additional metrics, install the [Datadog Google Cloud Platform integration][6].
+
+## Setup
+
+{{< programming-lang-wrapper langs="nodejs,python" >}}
+{{< programming-lang lang="nodejs" >}}
+1. **Install dependencies**. Run the following commands:
+   ```shell
+   npm install @datadog/serverless-compat
+   npm install dd-trace
+   ```
+
+   To use [automatic instrumentation][1], you must use `dd-trace` v5.25+.
+
+   Datadog recommends pinning the package versions and regularly upgrading to the latest versions of both `@datadog/serverless-compat` and `dd-trace` to ensure you have access to enhancements and bug fixes.
+
+2. **Start the Datadog serverless compatibility layer and initialize the Node.js tracer**. Add the following lines to your main application entry point file (for example, `app.js`):
+
+   ```js
+   require('@datadog/serverless-compat').start();
+
+   // This line must come before importing any instrumented module. 
+   const tracer = require('dd-trace').init()
+   ```
+
+3. (Optional) **Enable runtime metrics**. See [Node.js Runtime Metrics][2].
+
+4. (Optional) **Enable custom metrics**. See [Metric Submission: DogStatsD][3].
+
+[1]: /tracing/trace_collection/automatic_instrumentation/?tab=singlestepinstrumentation
+[2]: /tracing/metrics/runtime_metrics/nodejs/?tab=environmentvariables
```

Comment:
```suggestion
[2]: /tracing/metrics/runtime_metrics/?tab=nodejs
```

Reply:
Looks like these links have been consolidated onto tabs - these new links will let users jump straight to the relevant tab 🙂